pyramid_zipkin/tween.py

```diff
@@ -73,8 +75,14 @@ def _get_settings_from_request(request):
 
     if 'zipkin.transport_handler' in settings:
         transport_handler = settings['zipkin.transport_handler']
-        stream_name = settings.get('zipkin.stream_name', 'zipkin')
-        transport_handler = functools.partial(transport_handler, stream_name)
+        if not isinstance(transport_handler, BaseTransportHandler):
+            warnings.warn(
+                'Using a function as transport_handler is deprecated. '
+                'Please extend py_zipkin.transport.BaseTransportHandler',
+                DeprecationWarning,
+            )
+            stream_name = settings.get('zipkin.stream_name', 'zipkin')
+            transport_handler = functools.partial(transport_handler, stream_name)
     else:
         raise ZipkinError(
```

Comment:
GH doesn't allow me to put this comment 3 lines below, but can you change the `scribe/kafka` to something more generic?